drivers/storage/portworx/component/autopilot.go

```diff
@@ -171,6 +171,37 @@ func (c *autopilot) createConfigMap(
 			provider.Name, provider.Type, params)
 	}
 
+	if cluster.Spec.Autopilot.GitOps != nil {
+		config += "\ngitops:"
+
+		keys := make([]string, 0, len(cluster.Spec.Autopilot.GitOps.Params))
+		for k := range cluster.Spec.Autopilot.GitOps.Params {
+			keys = append(keys, k)
+		}
+		sort.Strings(keys)
+
+		params := ""
+		for _, key := range keys {
+			val := cluster.Spec.Autopilot.GitOps.Params[key]
+			if key == "defaultReviewers" {
+				params += `
+    defaultReviewers:`
+				for _, v := range val.([]string) {
```

Comment:
This will panic if the value is not an array of strings. We should check and fail properly with an error. Add a test for that.

Reply:
as it's not strong typed, you may want to use comma separated string.